src/App.tsx

```diff
@@ -1,61 +1,126 @@
 import './App.scss';
+import { useState } from 'react';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
+
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { Todo } from './types/TODO';
+
+const idForNewTodo = Math.max(...todosFromServer.map(todo => todo.id)) + 1;
+
+export function getUserById(userId: number) {
+  return usersFromServer.find(user => user.id === userId) || null;
+}
+
+export const App: React.FC = () => {
+  const initialTodoState = {
+    id: idForNewTodo,
+    title: '',
+    completed: false,
+    userId: 0,
+  };
+
+  const initialTodos = [...todosFromServer];
+
+  const [newTodo, setNewTodo] = useState(initialTodoState);
+  const [todos, setTodos] = useState(initialTodos);
+  const [hasTitleError, setHasTitleError] = useState(false);
+  const [hasUserIdError, setHasUserIdError] = useState(false);
```

Comment:
you can also create as one state (using an object) instead of two different ones